src/utils/viewer/viewer.cpp

```diff
@@ -899,7 +899,8 @@ Viewer::Viewer(const Arguments& arguments)
 
     Cr::Utility::Resource rs{"fonts"};
     font_ = fontManager_.loadAndInstantiate("TrueTypeFont");
-    if (!font_ || !font_->openData(rs.getRaw("ProggyClean.ttf"), fontSize))
+    std::string font_relative_path = "../../../data/fonts/ProggyClean.ttf";
+    if (!font_ || !font_->openData(rs.getRaw(font_relative_path), fontSize))
```

Comment:
... and this change then isn't needed.

Reply:
Do you know if I can do something similar in viewer.py to avoid hardcoding the ugly relative path?

Reply:
Is the `viewer.py` meant to be packaged / installed in some way (`setup.py`, pip etc.)? I know very little about Python packaging, but if it is, I'd put the font file into some "well-known package data location" (if there's such a thing) and reference it from there. If not, then I'm afraid the path has to be hardcoded that way. But I'd make it relative to `__file__`, at least, so it doesn't depend on CWD.

Another idea I had was that we embed the font directly inside the Habitat libraries instead of the C++ viewer, and expose that array to Python. Such as 

```cpp
Containers::ArrayView<const char> fontData() {
    Cr::Utility::Resource rs{"fonts"};
    return rs.getRaw("ProggyClean.ttf");
}
```

and then, if the above would be exposed as `habitat.font_data()` to Python, you'd do 

```py
self.display_font.open_data(habitat.font_data(), size)
```

instead of `self.display_font.open_file("that/ugly/path.ttf", size)`. A downside is that the binary now embeds a font file that's only needed if using a GUI viewer, but as long as we stay with that tiny 40 kB font file, I think the convenience wins over memory use :)